templates/hooks/snapshot/sdk_update_pre_build_request.go.tpl

```diff
@@ -0,0 +1,2 @@
+//Not used currently but whenever SDKUpdate() is generated for non existing update() operation on resource this will be added then as override
+return latest, nil
```

Comment:
You're probably safe to just delete this for now. If update isn't supported yet, better to return the not implemented error.